listenbrainz/webserver/__init__.py

```diff
@@ -407,3 +407,6 @@ def _register_blueprints(app):
     app.register_blueprint(album_bp, url_prefix='/album')
     app.register_blueprint(release_bp, url_prefix='/release')
     app.register_blueprint(release_group_bp, url_prefix='/release-group')
+    
+    from listenbrainz.webserver.views.atom import atom_bp
+    app.register_blueprint(atom_bp, url_prefix='/atom')
```

Comment:
Lets call the endpoint /feed rather than /atom -- atom is a technology name and it doesn't really inform the user that there are feeds available at this URL.

Reply:
Good point! I guess most of my feed subscriptions are technical writer and /atom isn't uncommon.

Reply:
Ah, I see, yes /feed can't work. How about /syndication-feed instead?

The reason to not use /atom is because that isn't considere to be RESTful:

  https://restfulapi.net/rest-api-design-tutorial-with-example/

A RESTful API structures its URLs in a way that describes the resources available, not the technology being used to generate the data or the format of the data being returned.